ts/Core/Chart/Chart.ts

```diff
@@ -2430,6 +2430,7 @@ class Chart {
                 { labels } = options;
 
             if (
+                chart.hasCartesianSeries && // #20948
```

Comment:
The fix seems fair, but can you shed some light on why it is needed? Intuitively I thought that if a map had an axis, it would be hidden, so that checking for `axis.visible`. Also, where does the axis come from? Is it the mapbubble series? Or may that be the actual bug, that this series type shouldn't have an axis?

Reply:
Previously, in HC v11.2.0 creation of axis groups was different and changed here: https://github.com/highcharts/highcharts/pull/19896 We need to create those groups to calculate `expectedSpace` for the axis, to change the plot dimensions. In HC v11.2.0 creation of groups wasn't there and now we have to prevent it for map charts.